src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -526,7 +526,8 @@ export default class DefaultAudioVideoController
         this.meetingSessionContext.videoUplinkBandwidthPolicy = new NScaleVideoUplinkBandwidthPolicy(
           this.configuration.credentials.attendeeId,
           !this.meetingSessionContext.browserBehavior.disableResolutionScaleDown(),
-          this.meetingSessionContext.logger
+          this.meetingSessionContext.logger,
+          this.meetingSessionContext.browserBehavior
```

Comment:
nit: trailing comma

Reply:
???

Reply:
nvm, the initialization of classes does not seem to apply this rule